test/test_library.py

```diff
@@ -730,6 +730,16 @@ def test_unique_sanitized(self):
         self._setf(u'foo%aunique{albumartist album,albumtype}/$title')
         self._assert_dest(b'/base/foo [foo_bar]/the title', self.i1)
 
+    def test_drop_empty_disam_string(self):
+        album1 = self.lib.get_album(self.i1)
+        album1.year = None
+        album1.store()
+        self._assert_dest(b'/base/foo/the title', self.i1)
```

Comment:
Hmm; I suppose I assumed you had run into this with some music in your library. Maybe you can reuse the data from there? You could, for example, try setting some arbitrary field `foo` to be empty on one album and to have a known value on another, and then set up the `%aunique` invocation to use that as the only (or first) disambiguator.